printTables.py

```diff
@@ -0,0 +1,247 @@
+
+
+
+"""
+   Discord has a max width of 80 characters until it word wraps. Therefore, 79 is the max word length
+   Create dynamic column sizes by creating an array for width of each column.
+
+   make maxWordLen =  max row size
+
+
+"""
+class OutputTable:
+
+
+    def __init__(self, matrixTable, maxRowWidth):
+        self.table = matrixTable
+        self.boarder = "|"
+        self.rowSym = "-"
+        self.intersectionChar = "+"
+        self.outputString = ""
+        self.tableSize = len(self.table)
+        self.headerSize = 0
+        self.getHeaderSize()
+        self.maxWordLen = int(maxRowWidth / self.headerSize)
+        self.maxRowWidth = maxRowWidth
+        self.columnWidth = []
+
+
+
+
+
+    def getHeaderSize(self):
+
+        if self.tableSize > 0:
+            self.headerSize = len(self.table[0])
+
+
+    """
+    Accessor function for the maximum word length variable 
+    """
+    def getMaxWordLength(self):
+
+        return self.maxWordLen
+
+    """
+    Mutator function maximum word length  variable 
+    """
+    def setMaxWordLength(self, newLen):
+
+        self.maxWordLen = newLen
+
+    """
+     Accessor function for the boarder character variable
+    """
+    def getBoarderChar(self):
+
+        return str(self.boarder)
+
+    """
+    Mutator function for the boarder character variable 
+    """
+    def setBoarderChar(self, newChar):
+
+        self.boarder = str(newChar)
+
+    def makeString(self, appendString):
+
+        self.outputString += appendString
+
+    """
+     Function is designed to multiply the member variable self.boarder to a specified number 
+     This function helps create the character designed boarder
+    """
+    def mulWordWithrowChar(self, num):
```

Comment:
I think this function can be updated to use maxRowWidth

Reply:
Ok. Yea this whole implementation was not really well thought out.  I have created a second version which seems more flexible and stable.